data-model/virtualcollections/v.0.1.0/schemas/virtualcollections.json

```diff
@@ -1 +1,115 @@
+          ]
+        },
+        "foaf:name": {
+          "type": "string",
+          "examples": ["My Name"],
+          "description": "Name of the creator."
+        }
+      },
+      "description": "The creator of the collection."
+    },
+    "ltc:ObjectGroup": {
+      "type": "object",
+      "properties": {
+        "ltc:conditionsOfAccess": {
+          "type": "boolean",
+          "examples": ["Open to the public"],
+          "description": "Information about who can access the collection being described or an indication of its security status."
+        },
+        "ltc:collectionName": {
+          "type": "string",
+          "examples": ["Arctiacaja1991Netherlands"],
+          "description": "A short title that summarizes the collection objects contained within the ObjectGroup."
+        },
+        "ltc:description": {
+          "type": "string",
+          "examples": ["A virtual collection representing all butterfly specimens in South Holland"],
+          "description": "A free text description or narrative about the collection."
+        },
+        "ltc:isKnownToContainTypes": {
+          "type": "array",
+          "description": "Flag property to indicate that the collection is known to include type specimens. `true` - this collection contains types; `false` - this collections does not contain types.",
+          "items": {
+            "type": "boolean",
+            "examples": ["true"],
+            "description": "Type known to be contained."
+          }
+        }
+      },
+      "description": "An intentionally grouped set of objects with one or more common characteristics."
+    },
+    "ltc:MeasurementOrFact": {
+      "type": "object",
+      "properties": {
+        "dwc:measurementType": {
+          "type": "string",
+          "examples": ["Object Count"],
+          "description": "The nature of the measurement, fact, characteristic, or assertion."
+        },
+        "dwc:measurementValue": {
+          "type": "number",
+          "examples": [100],
+          "description": "The numerical value of the measurement."
+        }
+      },
+      "description": "A measurement of or fact about a class within the standard, or a relationship between the ObjectGroup and an associated class."
+    },
+    "ods:search_filters": {
+      "type": "array",
+      "description": "Search filters selected for the virtual collection.",
+      "items": {
+        "type": "object",
+        "properties": {
+          "filter_name": {
+            "type": "string",
+            "examples": ["c4df6c19a91c2daaeccd9628e3d3f196"],
+            "description": "Name or identifier/UUID for the filter."
+          },
+          "filter_value": {
+            "type": "array",
+            "examples": ["topicDiscipline=Zoology&midsLevel=1&country=Indonesia&hasMedia=1"],
+            "description": "Value or configuration of the filter."
+          }
+        }
+      }
+    },
+    "ltc:basisOfScheme": {
```

Comment:
do we need this in our data model? we already have `ltc:description`

Reply:
I was debating this. Yes, description is enough but this from the ltc doc (https://tdwg.github.io/ltc/terms/index.html#LatimerCoreScheme_basisOfScheme) made me think that we could use it in a more FAIR/FDO Profile manner: "This property is intended to summarise the reason for grouping a number of ObjectGroups within the LatimerCoreScheme, and the purpose for which the data is intended to be used. This may also be reflected in the terms and metrics defined using the SchemeTerm and SchemeMeasurementOrFact classes respectively. Using this approach, standard and reusable profiles within the wider Latimer Core standard may be constructed for common collection descriptions use cases." We could implement this with controlled vocab. Examples given are these 
Accession
Inventory
Expedition
Digitisation planning
Collections assessment
Institution
Collections registry

Reply:
let's keep this for now.